mediapipe/tasks/ios/build_ios_framework.sh

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env bash
+# Copyright 2023 The MediaPipe Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Set the following variables as appropriate.
+#   * BAZEL: path to bazel. defaults to the first one available in PATH
+#   * FRAMEWORK_NAME: name of the iOS framework to be built. Currently the
+#   * accepted values are TensorFlowLiteTaskVision, TensorFlowLiteTaskText.
+#   * MPP_BUILD_VERSION: to specify the release version. defaults to 0.0.1-dev
+#   * IS_RELEASE_BUILD: set as true if this build should be a release build
+#   * ARCHIVE_FRAMEWORK: set as true if the framework should be archived
+#   * DEST_DIR: destination directory to which the framework will be copied
+
+set -ex
+
+if [[ "$(uname)" != "Darwin" ]]; then
+  echo "This build script only works on macOS."
+  exit 1
+fi
+
+BAZEL="${BAZEL:-$(which bazel)}"
+MPP_BUILD_VERSION=${MPP_BUILD_VERSION:-0.0.1-dev}
+MPP_ROOT_DIR=$(git rev-parse --show-toplevel)
+MEDIAPIPE_DISABLE_GPU=1
+
```

Comment:
It might be easier to parse if we add defaults for IS_RELEASE_BUILD, ARCHIVE_FRAMEWORK and DEST_DIR